nodejs/week2/meal-sharing/src/backend/api/meals-router.js

```diff
@@ -0,0 +1,33 @@
+const express = require("express");
+const router = express.Router();
+
+const meals = require("./../data/meals.json");
+
+router.get("/", async (request, response) => {
+  try {
+    console.log(meals);
+    console.log("in /api/meals");
+    const query = request.query;
+    if (query.maxprice){
+
+  //Parameters: maxPrice (Get meals that has a price smaller than maxPrice)
+  response.send(meals.filter(meal => meal.price <= query.maxPrice));
+    } else if (query.title) {
+
+  //Parameters: Title (Get meals that partially match a title)
+  response.send(meals.filter(meal => meal.title.toLowerCase().includes(query.title)));
```

Comment:
Good that you're using .toLowerCase() to compare two string variables. Don't forget though to use it for query.title as well, otherwise it won't work 100% correctly!